docs/src/guide/discretisation.jl

```diff
@@ -198,22 +199,121 @@ Nrange = 10:10:100
 plot(Nrange, abs.(fconv.(Nrange) .- fconv(200)); yaxis=:log, legend=false)
 
 # ### Exercise 2
-# !!! note "TODO"
-#     This solution has not yet been written. Any help with a PR is appreciated.
+#      ```math
+#         \bullet \langle e_G, e_{G'}\rangle = ∫_0^{2π} e_G^\ast(x) e_{G'}(x) d x = 1/2π ∫_0^{2π} e^i(G'-G)x d x
+#      ```
+# if G≠G', since the function is periodic over $[0, 2\pi]$:
+#      ```math
+#         \langle e_G, e_{G'}\rangle = \frac 1 {i2π(G'-G)}  ∫_0^{2π} (e^{ix})^{G'-G} d x = 0
+#      ```
+# if G=G':
+#     ```math
+#        \langle e_G, e_{G'}\rangle =  \frac 1 {2π} ∫_0^{2π} d x = 1
+#     ```
+# Therefore:
```

Comment:
Which function ? I think that's a bit wordy
```suggestion
# Since `e^{iy}` is a periodic function with period $2\pi$, $\int_0^{2\pi} e^{i m y} dy = \delta_{0,m}$. Therefore:
```